app/routes.py

```diff
@@ -1,2 +1,200 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify, make_response, request
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from datetime import datetime
+import requests
+from app import SLACK_TOKEN
 
+tasks_bp = Blueprint('tasks', __name__, url_prefix='/tasks')
+goals_bp = Blueprint('goals', __name__, url_prefix='/goals')
+
+@tasks_bp.route('', methods=['GET', 'POST'])
+def handle_tasks():
+        if request.method == 'GET':
+            sort_query = request.args.get("sort")
+            if sort_query == 'asc':
+                tasks = Task.query.order_by(Task.title.asc())
+            elif sort_query == 'desc':
+                tasks = Task.query.order_by(Task.title.desc())
+            else:
+                tasks = Task.query.all()
+            tasks_list = []
+            for task in tasks:
+                tasks_list.append({
+                    "id": task.task_id,
+                    "title": task.title,
+                    "description": task.description,
+                    "is_complete": False if task.completed_at is None else task.completed_at
+                })
```

Comment:
you can create a helper function or instance method in your task model to handle creating this dictionary. It will reduce redundancy in `routes.py`